rkt/cli_apps.go

```diff
@@ -163,3 +252,39 @@ func (ae *appExec) Type() string {
 }
 
 // TODO(vc): --mount, --set-env, etc.
```

Comment:
Maybe he meant that `--set-env` should also be per-app and should implement the flags.Value interface. Do you remember this comment,  @vcaputo?